acceptance/clients/clients.go

```diff
@@ -56,6 +60,7 @@ func AcceptanceTestChoicesFromEnv() (*AcceptanceTestChoices, error) {
 	shareNetworkID := os.Getenv("OS_SHARE_NETWORK_ID")
 	dbDatastoreType := os.Getenv("OS_DB_DATASTORE_TYPE")
 	dbDatastoreVersion := os.Getenv("OS_DB_DATASTORE_VERSION")
+	liveMigrate := os.Getenv("OS_LIVE_MIGRATE")
```

Comment:
It might be easier to do this:

```go
var liveMigrate bool
if v := os.Getenv("OS_LIVE_MIGRATE"); v != "" {
  liveMigrate = true
}
```

Reply:
@jtopjian Unfortunately I can't test it on my environment